ckan/requirements.txt

```diff
@@ -49,7 +49,7 @@ Mako==1.1.5
 Markdown==3.1.1
 MarkupSafe==2.0.1
 messytables==0.15.2
-newrelic==6.10.0.165
+newrelic==7.0.0.166
```

Comment:
It's the same increment on the minor release, even with the point release upgrade?

Reply:
Yep, it's a weird way to track builds... https://pypi.org/project/newrelic/#history